lib/gds_api/test_helpers/business_support_helper.rb

```diff
@@ -15,28 +15,27 @@ def setup_business_support_stubs(endpoint, path)
           else
             results = @stubbed_business_supports
           end
-          {:body => plural_response_base.merge("results" => results, "total" => results.size).to_json}
+          { body: plural_response_base.merge("results" => results, "total" => results.size).to_json }
         end
-
       end
 
-      def api_has_business_support(business_support, facets={})
+      def api_has_business_support(business_support, facets = {})
         facets = sanitise_facets(facets)
-        if business_support.is_a?(Symbol)
-          bs_with_facets = facets.merge(:title => business_support)
-        else
-          bs_with_facets = facets.merge(business_support)
-        end
+        bs_with_facets = if business_support.is_a?(Symbol)
```

Comment:
We've previously discussed disabling the Style/ConditionalAssignment cop that enforces this; it leads to very ugly results here and in a few cases below.